librz/core/linux_heap_glibc.c

```diff
@@ -538,6 +538,39 @@ void GH(print_heap_chunk)(RzCore *core) {
 	free(cnk);
 }
 
+/**
+ * \brief Prints compact representation of a heap chunk. Format: Chunk(addr=, size=, flags=)
+ * \param core RzCore pointer
+ * \param chunk Offset of the chunk in memory
+ */
+void GH(print_heap_chunk_simple)(RzCore *core, GHT chunk) {
+	GH(RzHeapChunk) *cnk = RZ_NEW0(GH(RzHeapChunk));
+	RzConsPrintablePalette *pal = &rz_cons_singleton()->context->pal;
+
+	if (!cnk) {
+		return;
+	}
+
+	(void)rz_io_read_at(core->io, chunk, (ut8 *)cnk, sizeof(*cnk));
+
+	PRINT_GA("Chunk");
+	rz_cons_printf("(addr=");
+	PRINTF_YA("0x%" PFMT64x, (ut64)chunk);
+	rz_cons_printf(", size=0x%" PFMT64x, (ut64)cnk->size & ~(NON_MAIN_ARENA | IS_MMAPPED | PREV_INUSE));
+	rz_cons_printf(", flags=");
+	if (cnk->size & NON_MAIN_ARENA) {
+		PRINT_RA("NON_MAIN_ARENA,");
```

Comment:
the `,` may be unnecessary. I suggest you keep track of whether `,` is necessary or not with a boolean variable and you print it before doing `PRINT_RA("IS_MAPPED")` and before `PRINT_RA("PREV_INUSE")`